models/src/main/java/com/vimeo/networking2/params/ModifyVideosInAlbumSpecs.kt

```diff
@@ -0,0 +1,23 @@
+package com.vimeo.networking2.params
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * An object that is used to patch video addition and deletion updates to an Album.
+ */
+@JsonClass(generateAdapter = true)
+data class ModifyVideosInAlbumSpecs(
+    /**
+     * The set of videos that should be removed.
+     */
+    @Json(name = "remove")
+    val removeVideoSet: Set<NamedWrapperForRemove>? = null,
+
+    /**
+     * The set of videos that should be added.
+     */
+    @Json(name = "set")
```

Comment:
Is this the correct JSON key? Just want to confirm since `ModifyVideoInAlbumsSpecs` has a similar property with the key as "add".

Reply:
I copied the original Java class, @howardrigberg maybe you can weigh in on why this is named `set` instead of `add`?